CondFormats/GeometryObjects/interface/PHGCalParameters.h

```diff
@@ -13,6 +13,13 @@ class PHGCalParameters {
 
   std::string name_;
   std::vector<double> cellSize_;
+  std::vector<double> slopeMin_;
```

Comment:
Checking: is it normal that the parameters definìned here between [L16](https://github.com/cms-sw/cmssw/pull/47999/files#diff-6f93c656440d324275fcd7f07127cdbed7886c3b689bf2ffdbea1b671f307492R16) and [L22](https://github.com/cms-sw/cmssw/pull/47999/files#diff-6f93c656440d324275fcd7f07127cdbed7886c3b689bf2ffdbea1b671f307492R22)  are the only ones not listed in classes_def-xml? If they are intended to be treated differently than the other class parameters, couldnt they be listed separately from the other ones?

Reply:
Missed out originally. **Now added**